dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/src/Audacia.CodeAnalysis.Analyzers/Rules/MethodLength/MethodLengthAnalyzer.cs

```diff
@@ -133,33 +147,40 @@ private static Location GetMemberLocation(ISymbol member, SemanticModel semantic
         private sealed class StatementWalker : CSharpSyntaxWalker
         {
             private CancellationToken _cancellationToken;
+
             private bool _nullChecksFinished;
 
+            private readonly SemanticModel _semanticModel;
+
             public int StatementCount { get; private set; }
 
-            public StatementWalker(CancellationToken cancellationToken)
+            public StatementWalker(SemanticModel semanticModel, CancellationToken cancellationToken)
             {
+                _semanticModel = semanticModel;
                 _cancellationToken = cancellationToken;
             }
 
             public override void Visit(SyntaxNode node)
             {
                 _cancellationToken.ThrowIfCancellationRequested();
 
-                if (IsStatement(node))
+                if (IsStatement(node, _semanticModel))
```

Comment:
⛏️ _nit:_ There is no need to pass `_semanticModel` as params into the private methods, you could access the field directly in the methods

Reply:
Re shuffled.